infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/CorfuReplicationDiscoveryService.java

```diff
@@ -720,6 +720,40 @@ public void processTopologyChangeNotification(DiscoveryServiceEvent event) {
         }
     }
 
+    /**
+     * Determine if any standby cluster has been added or removed from the topology
+     *
+     * @param discoveredTopology new discovered topology
+     * @return true, standby has been added or removed
+     * false, otherwise
+     */
+    private boolean isStandbyChange(TopologyDescriptor discoveredTopology) {
+        return topologyDescriptor.getStandbyClusters().keySet() != discoveredTopology.getStandbyClusters().keySet() ||
+                isNodeAddedOrRemoved(discoveredTopology.getStandbyClusters(), topologyDescriptor.getStandbyClusters());
+    }
+
+    /**
+     * Determine if any active cluster has been added or removed from the topology
```

Comment:
nit -> For correctness, we might need to change this for 'has been added, removed or modified'